docs/source/running-jobs/environment-variables.rst

```diff
@@ -77,40 +111,24 @@ For example, this is useful for passing secrets (see below) or passing configura
 See complete examples at `llm/vllm/serve.yaml <https://github.com/skypilot-org/skypilot/blob/596c1415b5039adec042594f45b342374e5e6a00/llm/vllm/serve.yaml#L4-L5>`_ and `llm/vicuna/train.yaml <https://github.com/skypilot-org/skypilot/blob/596c1415b5039adec042594f45b342374e5e6a00/llm/vicuna/train.yaml#L111-L116>`_.
 
 
-.. _passing-secrets:
-
-Passing secrets
-~~~~~~~~~~~~~~~~~~~~~~~~
-
-We recommend passing secrets to any node(s) executing your task by first making
-it available in your current shell, then using ``--env`` to pass it to SkyPilot:
-
-.. code-block:: console
-
-  $ sky launch -c mycluster --env WANDB_API_KEY task.yaml
-  $ sky exec mycluster --env WANDB_API_KEY task.yaml
-
-.. tip::
-
-   In other words, you do not need to pass the value directly such as ``--env
-   WANDB_API_KEY=1234``.
-
-
-
 
+.. _sky-env-vars:
 
 SkyPilot environment variables
 ------------------------------------------------------------------
 
-SkyPilot exports these environment variables for a task's execution. ``setup``
-and ``run`` stages have different environment variables available.
+SkyPilot exports several predefined environment variables for a task's execution, which
+are useful for frameworks that require information about the cluster or task, such as
+torch.distributed, OpenMPI, etc. See examples in :ref:`dist-jobs` and :ref:`managed-jobs`.
+
+``setup`` and ``run`` stages have different environment variables available:
```

Comment:
Should L234-240 be moved to this overview section?

Reply:
Done. Moved the lines.